frr-tools/cp-tool/cp-tool.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cp "github.com/otiai10/copy"
+
+	"github.com/go-kit/log/level"
+	"go.universe.tf/metallb/internal/logging"
+)
+
+func main() {
+	logger, err := logging.Init("error")
```

Comment:
we don't really need a logger here, let's just panic / fmt.Printf on the few occasions where needed

Reply:
logger removed.